src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -257,8 +257,10 @@ module private Read =
 module private Token =
     let create streamVersion : StreamToken =
         { value = box streamVersion
-          version = streamVersion
+          version = streamVersion + 1L
           streamBytes = -1 }
+    let inline version (token: StreamToken) = unbox<int64> token.value
```

Comment:
I think `streamVersion` is a better name - `version` in Equinox and the Cosmos and DDB stores is 0-based (naming is more terse as 1. Equinox is only concerned with streams 2. to differentiate it from StreamVersion/StreamRevision or other terms used by stores)
`streamVersion` in SSS/ESDB is -1 based